### PR TITLE
Fix remaining TXOP duration calculation

### PR DESCRIPTION
**Pull request by mgonzalezlopezudc**
## Summary

- fix `TxopProcedure::getRemaining()` to return the unspent TXOP duration
- preserve the existing zero clamp once the TXOP limit has expired
- add a focused unit test for timing boundaries and the TXOP lifecycle

## Root cause

While a TXOP was active, `getRemaining()` returned `simTime() - start`, which is the elapsed duration. Callers therefore received the spent portion of the TXOP as though it were the remaining transmission budget.

The corrected expression returns `start + limit - simTime()` before expiry and zero afterward.

## Impact

Code that uses the remaining TXOP duration can now make scheduling decisions from the actual available budget. The inactive-state behavior is unchanged: querying before the TXOP starts or after it ends still raises a runtime error.

## Validation

- `source ./setenv -q && make MODE=release -j$(nproc)`
- `source ./setenv -q && inet_run_unit_tests -m release -f 'Ieee80211TxopProcedure_1\.test'`

The focused test covers the inactive state, the full limit at startup, partial elapsed time, exact expiry, post-expiry clamping, and the state after ending the TXOP.

### CODE REVIEW

File: tests/fingerprint/mipv6-refactoring.csv [hunk 1]
```diff
@@ -10,7 +10,7 @@
 # MIPv6 example — the primary test
 /examples/ipv6/mipv6/,               -f omnetpp.ini -c Handover -r 0,                70s,            aa29-a8c5/~tNlb, PASS, wireless EthernetMac
 /examples/ipv6/mipv6/,               -f omnetpp.ini -c RouteOptimizationTwoCNs -r 0,  60s,            068b-23aa/~tNlb, PASS, wireless EthernetMac
-/examples/ipv6/mipv6roaming/,        -f omnetpp.ini -c Roaming -r 0,                 70s,            c8dc-27c2/~tNlb, PASS, wireless EthernetMac
+/examples/ipv6/mipv6roaming/,        -f omnetpp.ini -c Roaming -r 0,                 70s,            7ed8-bee3/~tNlb, PASS, wireless EthernetMac
```
[the remark this answers is not in the document]
Comment (mgonzalezlopezudc) [reply tests/fingerprint/mipv6-refactoring.csv:13]: **Root cause**: bookkeeping failure to the **July 802.11 on-wire serialization batch**, which **updated the duplicate `examples.csv` row’s data-sensitive fingerprint but omitted ``mipv6-refactoring.csv``**.

Historical isolation found the real cause:

- Commit 632e460559 reproduces c8dc-27c2.
- The sequence-control endian change still reproduces c8dc-27c2.
- Adding the association-ID allocation changes it exactly to 7ed8-bee3.
- The AP now writes allocated AIDs into association/reassociation responses at [Ieee80211MgmtAp.cc (line 266)](src/inet/linklayer/ieee80211/mgmt/Ieee80211MgmtAp.cc:266). Event count remains 59,408, proving this is serialized packet-content change, not changed scheduling.

The corresponding [`examples.csv` row (line 385)](tests/fingerprint/examples.csv:385) already contains the correct network fingerprints; tplx, ~tNl, and ~tND all passed. Its graphical tyf ingredient differed under fake-GUI execution, but that ingredient is unrelated and intentionally unreliable.

I also reran all 20 relevant QoS/HCF baselines across examples.csv and showcases.csv, including the dedicated TXOP showcase. Every baseline passed unchanged, so the TXOP fix requires no additional fingerprint updates.

[the remark this answers is not in the document]
Comment (mgonzalezlopezudc) [reply tests/fingerprint/mipv6-refactoring.csv:13]: @levy ready for your review and approval

**Comment by tabgab** [comment]
Hi Miguel,

@levy is on holiday this week. I expect he will get back to you when he
returns. Thanks for the contributions!

Gábor

**Comment by levy** [comment]
Thanks, simple enough for easy review!
